BioArchLinux/biolibc/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Bipin Kumar <bipin@ccmb.res.in>
+
+pkgname=biolibc
+pkgver=0.2.4
+pkgrel=1
+pkgdesc="High-performance, memory-efficient bioinformatics library"
+arch=('x86_64')
+url="https://github.com/auerlab/biolibc"
+license=('BSD-2')
+depends=('glibc')
+makedepends=('libxtend')
+source=("$pkgname-$pkgver.tar.gz::https://github.com/auerlab/$pkgname/archive/refs/tags/$pkgver.tar.gz")
+sha256sums=('57535bb4f18f98cbfd47707c60f8a66fa411403d7897be257985046d82a3e94b')
+
+build() {
+  cd "$pkgname-$pkgver"
+
+  #./configure --prefix=/usr
+  make PREFIX="$pkgdir/usr" clean depend
+}
+
+package() {
+  cd "$pkgname-$pkgver"
+  make PREFIX="$pkgdir/usr" MANPREFIX="$pkgdir/usr/share" clean depend install
+  strip -s "$pkgdir"/usr/lib/libbiolibc.so.2.0
```

Comment:
Is this really needed? stripping is done automatically after `package()`, unless you add `options=(!strip)` in PKGBUILD

Reply:
if i dont add strip i get this warning with namcap `libxtend W: ELF file ('usr/lib/libxtend.so.2.0') is unstripped`

Reply:
> if i dont add strip i get this warning with namcap `libxtend W: ELF file ('usr/lib/libxtend.so.2.0') is unstripped`

Ok, it's too strange.